### PR TITLE
making the header responsive

### LINKED ISSUE #35: [HTML, CSS] making the header responsive.

### PR DESCRIPTION
**Pull request by Yousef-Dall**
relates #35

### CODE REVIEW

File: universitiesPage/universities.html [hunk 1]
```diff
@@ -20,6 +20,16 @@
...
+
+                <ul id="list">
+                    <li><a href="../index.html">home</a></li>
+                    <li id="search"><a href="search.html">search</a></li>
+                    <li><a href="../aboutPage/about.html">about us</a></li>
+                </ul>
+    
+            </div>
+
+            <div id="menu">
```
Comment (yaqootturman) [inline universitiesPage/universities.html]: The button in the Universities page takes a different design
check it, please!

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: universitiesPage/universities.html (hunk 1 above)
Comment (nada-abuzaid) [reply universitiesPage/universities.html]: I already modified the design on the university page, check it when I upload the responsive design!

File: searchPage/search.css [hunk 2]
```diff
@@ -1,184 +1,277 @@
...
-  display: flex;
-  justify-content: space-between;
-  margin-top: 3.7vh;
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    margin-top: 3.7vh;
 }
+
 #logo {
```
Comment (yaqootturman) [inline searchPage/search.css:22]: You already add font-family: 'Roboto', sans-serif; in university but didn't add it for other pages (for logo class)

File: generalInfoPage/general.html [hunk 3]
```diff
@@ -25,6 +25,16 @@
...
+                <li><a href="../index.html">home</a></li>
+                <li id="search"><a href="search.html">search</a></li>
+                <li><a href="../aboutPage/about.html">about us</a></li>
+            </ul>
+
+        </div>
+
+          <div id="menu">
               <button onclick="show()">
                 <i id="menu-icon" class="fas fa-align-right fa-flip-vertical"></i>
```
Comment (yaqootturman) [inline generalInfoPage/general.html:39]: the menu icon didn't have the same style on all pages
in some, it misses font-size: 1.5rem;

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by basillubbad — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Review by Fady-Alwazir — APPROVED** [review]

**Pull request merged**